.github/workflows/ci.yaml

```
@@ -119,9 +119,9 @@ jobs:
         run: sbt ";dotty-bootstrapped/compile ;dotty-bootstrapped/test"
         shell: cmd
 
-      # - name: Scala.js Test
-      #   run: sbt ";sjsJUnitTests/test"
-      #   shell: cmd
+      - name: Scala.js Test
+        run: sbt ";sjsJUnitTests/test"
```

Perhaps also add
```suggestion
        run: sbt ";sjsJUnitTests/test ;sjsCompilerTests/test"
```
?